model_compression_toolkit/core/common/quantization/quantization_params_generation/lut_kmeans_params.py

```diff
@@ -51,10 +50,10 @@ def lut_kmeans_tensor(tensor_data: np.ndarray,
 
     Returns:
         A dictionary containing the cluster assignments according to the k-means algorithm,
-        the scales per channel and the multiplier num bits.
+        the thresholds per channel and the multiplier num bits.
     """
-    if n_bits > MULTIPLIER_N_BITS:
-        raise Exception(f'Look-Up-Table bit configuration has {n_bits} bits. It must be less or equal to {MULTIPLIER_N_BITS}')
+    if n_bits >= MULTIPLIER_N_BITS:
```

Comment:
Why the equal was added?

Reply:
I discussed it with Hai and we came to an understanding that 8-bit lut quantization is not practical (and sometimes can collapse the program due to memory issues).